Softeq.XToolkit.WhiteLabel.Forms/Navigation/FormsNavigationService.cs

```diff
@@ -77,7 +74,7 @@ public void GoBack()
             Navigation.PopAsync().FireAndForget(_logger);
```

Comment:
Seems a bit inconsistent that some of the navigation methods were made async, but some of them weren't.. maybe we should make all of them async?